### DISCUSSION TITLE
Is that possible bypassing cloudflare?

### DISCUSSION THREAD

**Discussion by SSujitX** [D4Vinci/Scrapling#19] [General]
`from scrapling.defaults import Fetcher, StealthyFetcher, PlayWrightFetcher

page = PlayWrightFetcher.fetch(
    "https://gitlab.com/users/sign_in",
    disable_resources=True,
    headless=False,
    stealth=True,
)

print(page.html_content)`

How can I bypass Cloudflare?

**Comment by D4Vinci** [comment]
Just use the StealthyFetcher bro and make it wait for the login form to show up as follows:
```
page = StealthyFetcher.fetch("https://gitlab.com/users/sign_in", headless=False, wait_selector='.form-group')
```
